openedx/adg/lms/applications/handlers.py

```diff
@@ -31,3 +34,30 @@ def delete_business_line_group(instance, **kwargs):  # pylint: disable=unused-ar
     """
     # TODO: Use instance.group.delete(), once LP-2479 is deployed on prod.
     Group.objects.filter(name=instance.title).delete()
+
+
+@receiver(post_save, sender=CourseEnrollment)
+def update_prereq_courses_status(instance, **kwargs):  # pylint: disable=unused-argument
+    """
+    Update prerequisite course groups status to IN_PROGRESS in ApplicationHub when user in enrolled in
+    prereq course.
```

Comment:
Can we better explain the functionality of the function (no pun intended) in the docstring? Something like:

"Update program_prereq_courses_status or bu_prereq_courses_status to IN_PROGRESS in Application Hub depending on the prereq course that the learner has enrolled in"